xds/src/test/java/io/grpc/xds/XdsClientImplTestBase.java

```diff
@@ -3567,11 +3567,8 @@ public void sendToNonexistentHost() throws Exception {
     // Setup xdsClient to fail on stream creation
     XdsClientImpl client = createXdsClient("some.garbage");
     client.watchXdsResource(XdsListenerResource.getInstance(), LDS_RESOURCE, ldsResourceWatcher);
-    fakeClock.forwardTime(20, TimeUnit.SECONDS);
-
     verify(ldsResourceWatcher, Mockito.timeout(5000).times(1)).onError(ArgumentMatchers.any());
-    fakeClock.forwardTime(50, TimeUnit.SECONDS); // Trigger rpcRetry if appropriate
-    assertThat(fakeClock.getPendingTasks(LDS_RESOURCE_FETCH_TIMEOUT_TASK_FILTER)).isEmpty();
+    assertThat(fakeClock.numPendingTasks()).isEqualTo(1); //retry
```

Comment:
As a nit, It would probably be good to verify that it is the expected retry task